utils/utils.go

```diff
@@ -350,15 +352,17 @@ func DecompressFile(inPath string) (outPath string, errRet error) {
 	return outPath, errRet
 }
 
-func GetFileSum(filepath string) (outSum []byte, errRet error) {
-	//nolint:gosec
-	in, err := os.Open(filepath)
+func GetFileSum(path string) (outSum []byte, errRet error) {
+	resolvedPath, err := filepath.EvalSymlinks(path)
```

Comment:
so we were just getting the checksum of the symlink before? and that's why file:// links weren't triggering a checksum mismatch and redownload?